tests/scenarios/watch-mode-test.ts

```diff
@@ -239,13 +226,115 @@ class EmberCLI {
   }
 }
 
+class File {
+  constructor(readonly label: string, readonly fullPath: string) {}
+
+  async exists(): Promise<boolean> {
```

Comment:
I think we have fs-extra available, idk if that'd help clean this up with pathExists?

Reply:
A tiny bit, but probably not enough to watch CI run again 🤷🏼‍♂️ if it fails for other reasons I'll refactor it, otherwise I'll do it when I fix the actual issue